test/bin/scenario.sh

```diff
@@ -139,17 +139,23 @@ sos_report_for_vm() {
     # Some scenarios do not start with MicroShift installed, so we
     # can't rely on the wrapper being there or working if it
     # is. Copy the script to the host, just in case, along with a
-    # wrapper that knows how to execute it or the installed
-    # version.
+    # wrapper that knows how to execute it or the installed version.
+    # Add anaconda to the list of the plugins to collect information
+    # about failed image installations.
     cat - >/tmp/sos-wrapper.sh <<EOF
 #!/usr/bin/env bash
+PLUGINS=container_log,crio,logs,rpmostree,rpm,anaconda
```

Comment:
Let's think if including `anaconda` in current state it the best action

Reply:
We need /var/log/anaconda/ directory contents for debugging the installation.
Should we just copy those explicitly?

Reply:
The problem seems to be in the /root/anaconda-ks.cfg file.
Let me try to copy the contents of /var/log/anaconda explicitly, without using the plugin.